### PR TITLE
chrome37 fixes (remove dead closing code), removed unused files, link to dev of uproxy-lib

### PR DESCRIPTION
**Pull request by iislucas**
- Some fixed for chrome37 (remove dead closing code)
- Removed unused freedom-module files
- Link to dev of uproxy-lib in package.json - dev should link to dev branches so we know when it breaks. 
- Adds proxy-config parameter to rtc-net for configuration limit of non-routable addresses

### CODE REVIEW

File: src/rtc-to-net/rtc-to-net.ts [hunk 1]
```diff
@@ -21,9 +21,19 @@ module RtcToNet {
 
   var log :Freedom_UproxyLogging.Log = freedom['core.log']('RtcToNet');
 
+  export interface ProxyConfig {
+    // If |alloNonUnicast === false| then any proxy attempt that results
+    // in a non-unicast (e.g. local network) address will fail.
+    alloNonUnicast :boolean;
```
Comment (salomegeo) [inline src/rtc-to-net/rtc-to-net.ts]: Is this supposed to be allowNonUnicast?

Comment (salomegeo) [inline src/rtc-to-net/rtc-to-net.ts]: Do you intend to put anything else in ProxyConfig?

Comment (iislucas) [inline src/rtc-to-net/rtc-to-net.ts]: yes. Thanks. fixed.

Comment (iislucas) [inline src/rtc-to-net/rtc-to-net.ts]: Yes, more things to appear in ProxyConfig in the future, e.g. patterns for white/blacklist of domains/IPs have been suggested. Also bandwidth controls.

**Comment by iislucas** [comment]
PTAL. Give the :+1: when you're happy with it. :)

**Comment by salomegeo** [comment]
:+1:

**Pull request merged**